src/test/java/com/wikia/webdriver/common/core/drivers/BrowserType.java

```diff
@@ -0,0 +1,60 @@
+package com.wikia.webdriver.common.core.drivers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.Browsers.AndroidBrowser;
+import com.wikia.webdriver.common.core.drivers.Browsers.ChromeBrowser;
+import com.wikia.webdriver.common.core.drivers.Browsers.DefaultBrowser;
+import com.wikia.webdriver.common.core.drivers.Browsers.FirefoxBrowser;
+import com.wikia.webdriver.common.core.drivers.Browsers.GhostBrowser;
+import com.wikia.webdriver.common.core.drivers.Browsers.HtmlUnitBrowser;
+
+import org.openqa.selenium.support.events.EventFiringWebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.support.events.EventFiringWebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)